public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchRequest.ts

```diff
@@ -62,7 +67,7 @@ export abstract class CloudWatchRequest {
 
   // get the value for a given template variable
   expandVariableToArray(value: string, scopedVars: ScopedVars): string[] {
-    const variableName = this.templateSrv.getVariableName(value);
+    const variableName = getVariableName(value);
```

Comment:
Could we use `getTemplateSrv` from `@grafana/runtime` here so we could still use `templateSrv.getVariableName` instead of creating a new export in `@grafana/data`?

This question applies to other parts of this PR too. Just wanted to get an idea of whether we could avoid adding another export to `@grafana/data` if we can get the templateSrv which already has this function.

Reply:
It's not actually on the `@grafana/runtime` `templateSrv` interface, `getVariableName` currently only exists on the `app/features/templating/template_srv` implementation that we were importing. I can add it to the `@grafana/runtime` `templateSrv` if that's neater, but I was worried about that messing up types for other implementations of `templateSrv` (if that's how that works in typescript). Or we can add the function to `@grafana/runtime` if that seems better.

Reply:
https://github.com/grafana/grafana/pull/65259, there was some discussion about exposing this in `@grafana/runtime` earlier this year, but it ended up being scrapped.

I think some options we have are:
1. Exposing it in `@grafana/runtime` (not sure if it would be accepted, but 🤷
2. Keeping this functionality locally in CloudWatch (it looks like we're the only users of `getVariableName` anyway so we could just have a util in the CloudWatch code
3. Find a way to not use `getVariableName`. It's only used in `CloudWatchRequest.ts` and `templateVariableUtils.ts` with the rest of the usages being mocks. We might be able to replace the usages with the existing functionality exposed on the API that `@grafana/runtime` has.

Reply:
I'd go with 2, since we're the only ones that use it, but we can also try 1 again :D

Reply:
Done! I went with 2